cmd/sks_create.go

```diff
@@ -206,6 +206,6 @@ func init() {
 	sksCreateCmd.Flags().Int64("nodepool-disk-size", 50,
 		"default Nodepool Compute instances disk size")
 	sksCreateCmd.Flags().StringSlice("nodepool-security-group", nil,
-		"default Nodepool Security Group <name | id> (can be specified multiple times)")
+		"default Nodepool Security Group NAME|ID|NAME|ID (can be specified multiple times)")
```

Comment:
Erroneous repetition?
```suggestion
		"default Nodepool Security Group NAME|ID (can be specified multiple times)")
```

Reply:
Yes, typo from me 😒